themes/digital.gov/layouts/_default/baseof.html

```diff
@@ -44,8 +44,8 @@
   <div id="clicks_iframe" class="display-none"></div>
 
   <div class="edit_tools">
-    <a class="edit edit-open" aria-label="edit" href="#"><i class="far fa-edit"></i></a>
-    <a class="edit edit-issue" target="_blank" href="https://github.com/GSA/digitalgov.gov/issues/new?body=%0D%0A%2A%2A{{- .Title -}}%2A%2A%0D%0Ahttps%3A%2F%2Fdigital.gov%2F{{- .Permalink -}}%0D%0A%0D%0A---%0D%0A%0D%0A%2A%2APlease+describe+the+issue+clearly.+Include+a+screenshot+if+needed%2A%2A%0D%0A"><i class="fas fa-bug"></i></a>
+    <button class="edit edit-open" aria-label="edit" href="#"><i class="far fa-edit"></i><span class="screenreaderonly">Toggle Edit this page</span></span></button>
```

Comment:
USWDS provides a class called `usa-sr-only` that performs the same function as this. So custom CSS might not be necessary.

Also this is a button, so the `href` is not necessary. You should `type="button"` though.

Reply:
Good to know about the USWDS class. 
The edit-issue is visually a button, but is actually takes the user to an external link on GitHub

Reply:
I got ya, you meant the href="#" in the button. Got it.